verifier/src/qe_identity.rs

```diff
@@ -612,12 +616,39 @@ mod test {
         );
     }
 
+    #[test]
+    fn qe_identity_verifier_passes_at_next_update_ignoring_time() {
+        let json = include_str!("../data/tests/qe_identity.json");
+        let signed_qe_identity =
+            SignedQeIdentity::try_from(json).expect("Failed to parse signed identity");
+        let key = qe_verifying_key();
+        let verifier = SignedQeIdentityVerifier::new(Some(key), None);
+
+        let verification = verifier.verify(&signed_qe_identity);
+
+        assert_eq!(verification.is_success().unwrap_u8(), 1);
+    }
+
+    #[test]
+    fn qe_identity_verifier_passes_before_issue_date_ignoring_time() {
+        let json = include_str!("../data/tests/qe_identity.json");
+        let signed_qe_identity =
+            SignedQeIdentity::try_from(json).expect("Failed to parse signed identity");
+        let key = qe_verifying_key();
+        let verifier = SignedQeIdentityVerifier::new(Some(key), None);
+
+        let verification = verifier.verify(&signed_qe_identity);
+
+        assert_eq!(verification.is_success().unwrap_u8(), 1);
+    }
+
```

Comment:
This looks exactly the same as the previous test, I don't think it's providing anything.
```suggestion
```

Reply:
Ah, yes, I think I got a little ahead of myself there. Indeed there is only one path to test here. I'll get rid of both duplicate tests. Good point.